### ISSUE TITLE
Cannot load Toolbox 7.2.0 on a fresh Win10

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/R-Toolbox#72]
Fresh Win10 + R 3.4 + RStudio + OSPS 7.2.2. Installed the 7.2.0 toolbox on R 3.4. Trying to load it fails with: 
```
Error: package or namespace load failed for ‘MoBiToolboxForR’ in inDL(x, as.logical(local), as.logical(now), ...):
 unable to load shared object 'C:/Users/***/Documents/R/win-library/3.4/MoBiToolboxForR/libs/i386/DCIR6_0.dll':
  LoadLibrary failure:  The specified module could not be found.
```
The library is there, R is x86.

**Comment by msevestre** [comment]
Can you check if 7.1 works??
We should also check that the newest version works. Could you try it?

**Comment by PavelBal** [comment]
I compiled a version of the toolbox with the newest SimModel and Toolbox-code and it works in R 3.5.

**Comment by msevestre** [comment]
@PavelBal Is this version compatible with 3.4 as well? In that case I could see that we update soon. But otherwise this will probably have to wait

**Comment by PavelBal** [comment]
No, 7.1 does not work neither. My guess is the problem with C++ redist - could it be that Toolbox 7.2.0 (with DCI6_0) needs other redist than included in OSPS 7.2.2 (with DCI6_1)?

**Comment by msevestre** [comment]
@Yuri05 DCI6_0 and DCI6_1 all under 7.2.0? What did we mess up here? :)

**Comment by PavelBal** [comment]
> DCI6_0 and DCI6_1 all under 7.2.0? What did we mess up here? :)

DCI6_1 is not in the release of 7.2.0, but the master-branch of SimModel has 6_1. I guess I confused you - OSPS release 7.2.2 still has the 7.2.0 R-Toolbox with DCI6_0. The newest code of SimModel has DCI6_1, and with that I can compile a package that works.

**Comment by Yuri05** [comment]
Fixed in 7.3
